src/ImpactX.H

```diff
@@ -9,10 +9,14 @@
 
 #include "particles/ImpactXParticleContainer.H"
 
+#include "particles/elements/Drift.H"
+
 #include <AMReX_AmrCore.H>
 
 #include <memory>
 
+#include <list>
```

Comment:
Just grouping and sorting includes:
```suggestion
#include <list>
#include <memory>

```